examples/hosts/app-integration/external-data/README.md

```diff
@@ -0,0 +1,49 @@
+# @fluid-example/app-integration-external-data
+
+This example demonstrates how data from an external data source (e.g. a work tracking system) might be integrated with Fluid to enable more-real-time collaboration.  For example, to allow collaborators to see proposed changes updating live before committing them back to the database.
+
+Please note that the ideas explored here are experimental and under development.  They are not yet recommended for broad use in production.
+
+## Scenario
+
+TBD
+
+## Strategy overview
+
+TBD
+
+## Generalizing to other data sources
+
+TBD
+
+<!-- AUTO-GENERATED-CONTENT:START (GET_STARTED:tinylicious=true) -->
+<!-- This section is automatically generated.
+To update it, edit docs/md-magic.config.js  then run 'npm run build:md-magic' in the docs folder. -->
+
+## Getting Started
+
+You can run this example using the following steps:
+
+1. Run `npm install` and `npm run build:fast -- --nolint` from the `FluidFramework` root directory.
+    - For an even faster build, you can add the package name to the build command, like this:
+      `npm run build:fast -- --nolint @fluid-example/app-integration-external-data`
+1. In a separate terminal, start a Tinylicious server by following the instructions in [Tinylicious](../../../server/tinylicious).
```

Comment:
```suggestion
2. In a separate terminal, start a Tinylicious server by following the instructions in [Tinylicious](../../../server/tinylicious).
```

Reply:
This is actually just a little markdown trick - if you use 1. for everything, it will still render with correct numbers.  Makes it a little more convenient if later you add more items to the middle or rearrange them, so you don't have to manually renumber.